Zend/zend_fibers.h

```diff
@@ -92,9 +92,16 @@ static const zend_uchar ZEND_FIBER_STATUS_BAILOUT   = 0x20;
 
 static const zend_uchar ZEND_FIBER_STATUS_FINISHED  = 0x2c;
 
+/* These functions create and manipulate a Fiber object, allowing any internal function to start, resume, or suspend a fiber. */
+ZEND_API zend_fiber *zend_fiber_create(const zend_fcall_info *fci, const zend_fcall_info_cache *fci_cache);
+ZEND_API void zend_fiber_start(zend_fiber *fiber, zval *params, uint32_t param_count, zend_array *named_params, INTERNAL_FUNCTION_PARAMETERS);
```

Comment:
Like nikita said, don't use internal function params, pass what you need explicitly - retval and ex, retval should be last by convention.

Reply:
I'm not 100%, but I think it shouldn't be necessary to ever pass a frame pointer, I think EG(current_execute_data) will always be equal to execute_data at these boundaries.

Reply:
Yes, really should always be EG(current_execute_data), so I'll use that instead of a param and clean up the internal function params usage.